extensions/wikia/ContentReview/ContentReviewHelper.php

```diff
@@ -354,9 +354,25 @@ public function replaceWithLastApproved( \Title $title, $contentType, &$text ) {
 		}
 	}
 
+	/**
+	 * Checks if a user can edit a JS page in the MediaWiki namespace.
+	 * @param \Title $title
+	 * @param \User $user
+	 * @return bool
+	 */
 	public function userCanEditJsPage( \Title $title, \User $user ) {
 		return $title->inNamespace( NS_MEDIAWIKI )
 			&& $title->isJsPage()
 			&& $title->userCan( 'edit', $user );
 	}
+
+	/**
+	 * Checks if a user is a reviewer entitled to an automatic approval and if he requested it.
+	 * @param \User $user
+	 * @return bool
+	 */
+	public function userCanAutomaticallyApprove( \User $user ) {
+		return $user->isAllowed( 'content-review' )
+			&& $user->getRequest()->getBool( 'wpApprove' );
```

Comment:
Can we add explanation that wp prefix refers to WikiaPage and wpApprove is a checkbox near publish (wpSave) button on edit page?

Maybe it's not very universal description but explains naming pick and place of use. Might save some time of searching the code next time.